package.json

```diff
@@ -3,6 +3,7 @@
   "private": true,
   "dependencies": {
     "@rails/actioncable": "6.0.0-rc1",
+    "@rails/activestorage": "^6.0.0-alpha",
```

Comment:
This should be changed to `6.0.0-rc1`.  For some reason on npm it shows the latest release as alpha but if you go to all versions then there is a pre verion with rc1.